packages/pn-pa-webapp/src/redux/dashboard/__test__/reducers.test.ts

```diff
@@ -66,13 +64,9 @@ describe('Dashboard redux state tests', () => {
       status: '',
```

Comment:
Remove the empty parameters

Reply:
Same as what i said before, this can't be left empty because of the real API request